### PR TITLE
add CheckPotentialFoldersFilesExclusions for local sources version

### PR DESCRIPTION
**Pull request by pisatus**
This script allows to check potential excludable files and folders, not only by naming patterns but also by the contents of the file itself. 
It searches files with comments such as "Copyright" or "Licensed" and suggests them as 3rd party.

### CODE REVIEW

**Comment by nleach999** [comment]
@pisatus please integrate the changes into the existing version of the script rather than forking it and renaming it.  I would suggest making deep inspection of the file contents an optional task considering it could take a significant amount of time to perform in some cases.

Maintaining two scripts with overlapping features is not really feasible in the long term.

**Comment by pisatus** [comment]
> @pisatus please integrate the changes into the existing version of the script rather than forking it and renaming it. I would suggest making deep inspection of the file contents an optional task considering it could take a significant amount of time to perform in some cases.
> 
> Maintaining two scripts with overlapping features is not really feasible in the long term.

thanks for the review!
I made the changes accordingly! 
Please review

File: CheckPotentialFoldersFilesExclusions/README.md [hunk 1]
```diff
@@ -51,4 +51,7 @@ REMARKS
 
     For more information, type: "get-help C:\path\CheckExclusionsFoldersFiles.ps1 -detailed".
     
-    For technical information, type: "get-help C:\path\CheckExclusionsFoldersFiles.ps1 -full".
\ No newline at end of file
+    For technical information, type: "get-help C:\path\CheckExclusionsFoldersFiles.ps1 -full".
+
+    For local storage usage also checkout https://checkmarx.atlassian.net/wiki/spaces/AA/pages/6825215196/Quick+Exclusions
```
Comment (nleach999) [inline CheckPotentialFoldersFilesExclusions/README.md:56]: Please remove links to private URLs.

**Review by nleach999 — CHANGES_REQUESTED** [review]
Looks good to merge, but please remove private links.

File: CheckPotentialFoldersFilesExclusions/README.md (hunk 1 above)
Comment (pisatus) [reply CheckPotentialFoldersFilesExclusions/README.md]: hi, removed the line in question!

**Comment by pisatus** [comment]
Hi
I made the fixes suggested. 
thanks for all inputs!

**Review by nleach999 — APPROVED** [review]

**Pull request merged**